GenumKit/Parsers/StoryboardParser.swift

```diff
@@ -87,3 +97,8 @@ public final class StoryboardParser {
     }
   }
 }
+
+extension StoryboardParser.Segue: Equatable { }
+func ==(lhs: StoryboardParser.Segue, rhs: StoryboardParser.Segue) -> Bool {
+  return lhs.segueID == rhs.segueID
```

Comment:
Feels strange to only compare `segueID`.

What if the user wants to create a template that use both `segueUD` and `customClass` to generate the `case` name?

Like some might prefer to have their template generating this: `case {{segue.identifier|swiftIdentifier}}_{{segue.customClass|swiftIdentifier}} = {{segue.identifier}}` and thus if two segues have the same identifier but different customClass that wouldn't generate a conflict, and the user won't have the code he expect to be generated.